### PR TITLE
Fix relation permissions

### PR DESCRIPTION
**Pull request by NyanHelsing**

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -101,6 +102,25 @@ class CollectionViewSet(ModelViewSet):
     queryset = Collection.objects.all()
     serializer_class = CollectionSerializer
 
+    # TODO
+    # Monkeypatch fields from view so permission logic stays in the view
+
+    #protected_fields = [
```
Comment (cameronblandford) [inline api/views.py:108]: Should these be here?

Comment (NyanHelsing) [reply api/views.py:108]: I'd like them here, so people reading the code see it as a reminder. (Tickets get lost). This is also a starting point for how this bit of functionality would work, and it's probably easier to grok in situ as well. What do you think?

Comment (cameronblandford) [reply api/views.py:108]: that makes sense!

Comment (cameronblandford) [reply api/views.py]: maybe include a note to that effect, that this would be a starting place if people wanted to do that?

**Pull request merged**